pages/kids/stories/index.tsx

```diff
@@ -27,14 +28,27 @@ export interface Story {
   country: string;
 }
 
+const getStoriesByCountry = (stories: Story[]) => {
```

Comment:
Why do you need to create a dictionary data structure here? You are just converting it back to arrays anyway. 

Why not just 

```
const storiesSK = stories.filter(story => story.countries.includes("sk"))
```

You can adjust the JSON so that `story.countries` is an array.

Reply:
updated, based on discussion in Slack, we will have two groups "{jazyk} ludove rozpravky" and "ukrajinske ludove ..."